chargebee/src/main/java/com/chargebee/android/models/ChangeProductParams.kt

```diff
@@ -0,0 +1,7 @@
+package com.chargebee.android.models
+
+data class ChangeProductParams(
+    val purchaseProductParams: PurchaseProductParams,
+    val oldProductId: String,
+    val prorationMode: Int?
```

Comment:
Do we need the `prorationMode` right now, given we only support `IMMEDIATE_WITH_TIME_PRORATION` currently?
Does it make sense to compose `ChangeProductParams` with `PurchaseProductParams` as opposed to using individual parameters?

Reply:
done